### ISSUE TITLE
[Accessibility Insights] An element of the given ControlType must support the Text pattern for edit controls: TextBox/MaskedTextBox/ComboBox/NumericUpDown/DomainUpDown

### ISSUE THREAD

**Issue by Marie-Shi** [dotnet/winforms#1588]
* .NET Core Version: .NET Core 3.0.100-preview8-013656 from release branch
* Have you experienced this same bug with .NET Framework? Yes

**Affected controls:**
TextBox/MaskedTextBox/ComboBox/NumericUpDown/DomainUpDown

**Problem description:**
1.	Install the .NET Core 3.0 SDK from https://github.com/dotnet/core-sdk.
2.	Open the attached ‘NumericUpDown_DomainUpDown’ .NET Core app in VS.
[NumericUpDown_DomainUpDown.zip](https://github.com/dotnet/winforms/files/3480418/NumericUpDown_DomainUpDown.zip)
3.	Build and run it.
4.	Start Accessibility Insights for Windows and test NumericUpDown and DomainUpDown controls in Form.

**Actual behavior:** 
An element of the given ControlType doesn’t support the Text pattern. 
Violation: [Section 508 502.3.10](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology) 
Please see the below Screenshot:
NumericUpDown:
![NumericUpDown](https://user-images.githubusercontent.com/45864017/62684112-8d93ab00-b9f2-11e9-819b-a3519503fcd7.png)

DomainUpDown:
![DomainUpDown](https://user-images.githubusercontent.com/45864017/62684118-91273200-b9f2-11e9-94f5-3ed3f13f8644.png)

**Expected behavior:**
NumericUpDown and DomainUpDown eidt should support the Text pattern.

[EDIT]

- [x] `TextBox` #2701
- [x] `MaskedTextBox`  #2701
- [x] `ComboBox` #2701
- [x] `NumericUpDown` #3878
- [x] `DomainUpDown` #3878

**Comment by zsd4yr** [comment]
@M-Lipin assigning to you

**Comment by merriemcgaw** [comment]
Just need to get this ported to 6.

**Comment by Marie-Shi** [comment]
Verified this issue with .Net 6.0.100-alpha.1.20506.3 from master branch, it is fixed. Now those controls(TextBox, MaskedTextBox, ComboBox, NumericUpDown, DomainUpDown, RichTextBox) support support the Text pattern.
![Untitled](https://user-images.githubusercontent.com/45864017/95566010-95fed000-0a53-11eb-8336-631482d35920.png)
